CMakeLists.txt

```diff
@@ -8,6 +8,16 @@ project(KinesisVideoProducerCpp VERSION 3.4.0)
 set(CMAKE_CXX_STANDARD 11)
 include(GNUInstallDirs)
 
+set(KINESIS_VIDEO_PRODUCER_CPP_MAJOR_VERSION 3)
+set(KINESIS_VIDEO_PRODUCER_CPP_MINOR_VERSION 1)
```

Comment:
We should change this to 4 given our current version is 3.4.1

Reply:
+1